src/status_im2/contexts/chat/composer/sub_view.cljs

```diff
@@ -27,12 +25,8 @@
   [:f> f-blur-view layout-height focused?])
 
 (defn- f-shell-button
-  [{:keys [maximized? focused?]} {:keys [height]} {:keys [reply edit]}]
+  [{:keys [focused?]}]
   (let [customization-color (rf/sub [:profile/customization-color])
-        insets              (safe-area/get-insets)
-        extra-height        (utils/calc-top-content-height reply edit)
-        neg-y               (utils/calc-shell-neg-y insets maximized? extra-height)
-        y-container         (reanimated/use-shared-value neg-y)
```

Comment:
Checking if reply or edit not needed?

Reply:
As jump-to button is now part of sheet itself, it gets pushed up according to the size of sheet. We don't need to manually calculate position anymore.

<img src="https://github.com/status-im/status-mobile/assets/17097240/9bb60b1c-3d7a-4cf6-97d0-504595c8ef04" height=200/>